catalyst_voices/packages/catalyst_voices_blocs/lib/src/registration/cubits/wallet_link_cubit.dart

```diff
@@ -16,12 +19,17 @@ abstract interface class WalletLinkManager {
 
   void selectRoles(Set<AccountRole> roles);
 
-  void submitRegistration();
+  Future<void> prepareRegistration(SeedPhrase seedPhrase);
+
+  Future<void> submitRegistration();
 }
 
 final class WalletLinkCubit extends Cubit<WalletLinkStateData>
     implements WalletLinkManager {
-  WalletLinkCubit() : super(const WalletLinkStateData());
+  final TransactionConfigRepository txConfigRepository;
```

Comment:
Does it make sense to rename repo to `TxConfigRepository` or property do `transactionConfigRepository`

Reply:
I renamed the property. Most likely this repository's name isn't final. There will be some parameters related to Cardano's blockchain that we would need to fetch either from our backend or from external service. So it won't be just a transaction config but other types of configs as well. Though I'm not sure what exactly that will be so I'll keep the current name